packages/skills/carpark_detection/handlers.py

```diff
@@ -238,7 +238,7 @@ def _handle_inform(self, msg: FIPAMessage, dialogue: Dialogue) -> None:
                                                                                              tx_digest))
             proposal = cast(Description, dialogue.proposal)
             total_price = cast(int, proposal.values.get("price"))
-            is_settled = self.context.ledger_apis.is_tx_settled('fetchai', tx_digest, total_price)
+            is_settled = self.context.ledger_apis.is_tx_settled('fetchai', tx_digest)
```

Comment:
can we add a `TODO: check the tx_digest references a transaction with the correct terms`?